src/Microsoft.ML.Core/CommandLine/CmdParser.cs

```diff
@@ -495,6 +500,11 @@ private static ArgumentInfo GetArgumentInfo(Type type, object defaults)
 
                     if (map.ContainsKey(name.ToLowerInvariant()))
                         throw Contracts.Except("Duplicate name '{0}' in argument type '{1}'", name, type.Name);
+                    if (attrName != null)
+                    {
+                        if (map.ContainsKey(attrName.ToLowerInvariant()))
+                            throw Contracts.Except("Duplicate name '{0}' in argument type '{1}'", attrName, type.Name);
```

Comment:
Please convert these to be interpolated.

Reply:
Done